tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -88,10 +88,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
+            used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
+            log.info(f"Used size after IO in {filename} is {used_size_after_io}")
+            used_size_before_io = used_size_after_io
```

Comment:
What is the purpose of this variable`used_size_before_io`?
Do we use it elsewhere?

Reply:
> What is the purpose of this variable`used_size_before_io`? Do we use it elsewhere?
"used_size_before_io" is the size of the pool before the actual ios that test executes.

Reply:
> Do we use it elsewhere?

Yes, we use this variable every time to calculate the pool size after running ios on each file. Please look at line no 92.